Pt_E_N_Ht_Date:
- Point ID
- Easting
- Northing
- Elevation
- Date
- Time
- DistTime
